scripts/prepdocslib/searchmanager.py

```diff
@@ -48,23 +50,41 @@ def __init__(
         search_info: SearchInfo,
         search_analyzer_name: Optional[str] = None,
         use_acls: bool = False,
+        use_int_vectorization: bool = False,
         embeddings: Optional[OpenAIEmbeddings] = None,
         search_images: bool = False,
     ):
         self.search_info = search_info
         self.search_analyzer_name = search_analyzer_name
         self.use_acls = use_acls
+        self.use_int_vectorization = use_int_vectorization
         self.embeddings = embeddings
         self.search_images = search_images
 
-    async def create_index(self):
+    async def create_index(self, vectorizers: Optional[List[VectorSearchVectorizer]] = None):
         if self.search_info.verbose:
             print(f"Ensuring search index {self.search_info.index_name} exists")
 
         async with self.search_info.create_search_index_client() as search_index_client:
             fields = [
-                SimpleField(name="id", type="Edm.String", key=True),
-                SearchableField(name="content", type="Edm.String", analyzer_name=self.search_analyzer_name),
+                (
+                    SimpleField(name="id", type="Edm.String", key=True)
+                    if not self.use_int_vectorization
```

Comment:
This update will fail if the index already exists - document that int vectorization cannot be enabled on an existing index unless the allowIndexDowntime option is enabled? You also might want to allow passing this in as an additional arg in case customers want to opt-into mixing prepdocs and int vectorization

Reply:
I found out that hybrid operation is bit tricky. We can have a follow up PR once the skeletion is laid out.